cmake/thisdd4hep.sh

```diff
@@ -63,6 +65,10 @@ dd4hep_parse_this ${BASH_ARGV[0]} DD4hep;
 #
 # These 3 are the main configuration variables: ROOT, Geant4 and XercesC
 # --> LCIO & Co. are handled elsewhere!
+
+if [ -z $ROOTSYS ]; then
+    export ROOTSYS=`dirname @ROOT_DIR@`
```

Comment:
This might not be ok, since this can be actually $ROOTSYS/cmake. When you do cmake -DROOT_DIR=.... this can  be absolute path to the ROOTConfig.cmake